interface/patient_file/summary/weno_fragment.php

```diff
@@ -0,0 +1,85 @@
+<?php 
+
+/**
+ * Weno Fragment.
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @author    Kofi Appiah <kkappiah@medsov.com>
+ * @copyright Copyright (c) 2023 Kofi Appiah <kkappiah@medsov.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require "../../globals.php";
+
+use OpenEMR\Common\Acl\AclMain;
+use OpenEMR\Common\Twig\TwigContainer;
+
+$twig = new TwigContainer(null, $GLOBALS['kernel']);
+
+if(!AclMain::aclCheckCore('patients', 'med')){
+        exit;
```

Comment:
rec putting following line here (above the exit; command) to give a message that can not access this:
```
echo (new TwigContainer(null, $GLOBALS['kernel']))->getTwig()->render('core/unauthorized.html.twig', ['pageTitle' => xl("Prescriptions")]);
```